### PR TITLE
chore(PAN-2367): adding release ci

### PR DESCRIPTION
**Pull request by abeaudoinNexapp**

### CODE REVIEW

File: .github/workflows/on-merge-on-main-ci.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
 # This is a basic workflow to help you get started with Actions
 
-name: CI
+name: Merge-On-Main
```
Comment (abeaudoinNexapp) [inline .github/workflows/on-merge-on-main-ci.yml:3]: This is so very incredible 🥇

**Review by abeaudoinNexapp — COMMENTED** [review]
You did a not too meh job

**Comment by drichard-nexapp** [comment]
WAZZZUP

**Comment by Tristan-0D** [comment]
Everybody in the house say AAAAAYOOOOO

File: .github/workflows/on-release.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  release:
+    types: [created]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
```
Comment (Tristan-0D) [inline .github/workflows/on-release.yml:12]: AYOOOOOOO

**Comment by hsebouai** [comment]
Yooo, I don't like this, don't ship it XD

File: .github/workflows/on-release.yml (hunk 2 above)
Comment (abeaudoinNexapp) [reply .github/workflows/on-release.yml:12]: Fine then

**Review by Tristan-0D — APPROVED** [review]

**Pull request merged**